test/fixtures/firestore_fixtures.clj

```diff
@@ -0,0 +1,20 @@
+(ns fixtures.firestore-fixtures
+  (:require
+    [avisi.atlassian.connect.firestore :as firestore]
+    [mount.core :as mount :refer [defstate]]
+    [clojure.test :refer :all])
+  (:import
+    (com.google.cloud.firestore FirestoreOptions FirestoreOptions$Builder Firestore)))
+
+(defstate service
+  :start
+    ^Firestore
+    (.getService
+      ^FirestoreOptions
+      (->
+        (FirestoreOptions/newBuilder)
+        ^FirestoreOptions$Builder (.setProjectId "dev")
+        (.build)))
+  :stop (.close ^Firestore service))
+
+(defn with-test-firestore [f] (mount/start-with {#'avisi.atlassian.connect.firestore/service service}) (f))
```

Comment:
Ik zou liever zien dat we firestore in de env stoppen en dit dan kunnen mocken door dit mee te geven